sdks/python/apache_beam/transforms/display_test.py

```diff
@@ -22,14 +22,15 @@
 import unittest
 from datetime import datetime
 
+import six
+
```

Comment:
Nit: Lint is complaining of the ordering of this import:

```
+import hamcrest as hc
 import six
-
-import hamcrest as hc
 from hamcrest.core.base_matcher import BaseMatcher
 ````

Reply:
This one complains when the hamcrest imports are split but also complains when they are put together.  The desired order is also different than iSort.  A bit difficult to find the correct order.